src/helpers/3box.ts

```diff
@@ -1,14 +1,63 @@
 import { subgraphRequest } from '@snapshot-labs/snapshot.js/src/utils';
 
 export async function getProfiles(addresses) {
-  return subgraphRequest('https://api.3box.io/graph', {
-    profiles: {
-      __args: {
-        ids: addresses
-      },
-      name: true,
-      eth_address: true,
-      image: true
+  const [ensProfiles, threeBoxProfiles] = await Promise.all([

```

Comment:
There should be a try catch here instead of having it on web3.ts. And if the request fail it should respond with an object of empty profiles like this:
```json
{
   "0x123": {},
   "0x345": {}
}
```
You can create the object first like this:
```js
const profiles = Object.fromEntries(addresses.map(address => [address, {}));
```